engine/src/test/java/io/zeebe/engine/processing/incident/JobFailIncidentTest.java

```diff
@@ -123,20 +123,12 @@ public void shouldCreateIncidentIfJobHasNoRetriesLeft() {
             .withIntent(ProcessInstanceIntent.ELEMENT_ACTIVATED)
             .getFirst();
 
-    final Record incidentCommand =
-        RecordingExporter.incidentRecords()
-            .withIntent(IncidentIntent.CREATE)
-            .withProcessInstanceKey(processInstanceKey)
-            .getFirst();
-
     final Record<IncidentRecordValue> incidentEvent =
         RecordingExporter.incidentRecords()
             .withIntent(IncidentIntent.CREATED)
             .withProcessInstanceKey(processInstanceKey)
             .getFirst();
 
-    assertThat(incidentCommand.getSourceRecordPosition())
-        .isEqualTo(failedEvent.getSourceRecordPosition());
```

Comment:
I think we can verify the source record position of the incidentEvent instead. (please also note the other test case)